pkg/cmd/server/bootstrappolicy/securitycontextconstraints_test.go

```diff
@@ -25,11 +27,20 @@ func TestBootstrappedConstraints(t *testing.T) {
 	groups, users := GetBoostrapSCCAccess(DefaultOpenShiftInfraNamespace)
 	bootstrappedConstraints := GetBootstrapSecurityContextConstraints(groups, users)
 
-	if len(expectedConstraints) != len(bootstrappedConstraints) {
-		t.Errorf("unexpected number of constraints: found %d, wanted %d", len(bootstrappedConstraints), len(expectedConstraints))
+	if len(expectedConstraintNames) != len(bootstrappedConstraints) {
+		t.Errorf("unexpected number of constraints: found %d, wanted %d", len(bootstrappedConstraints), len(expectedConstraintNames))
 	}
 
+	var orderedBootstrappedConstraints []*securityapi.SecurityContextConstraints
```

Comment:
Because `ByPriority` wants pointers and `bootstrappedConstraints` does not get pointers from `GetBootstrapSecurityContextConstraints`.

I tried  `sort.Sort(scc.ByPriority(bootstrappedConstraints))` but that did not pan out.